test/functional/readpreference.test.js

```diff
@@ -250,7 +263,9 @@ describe('ReadPreference', function () {
 
     test: function (done) {
       var configuration = this.configuration;
-      var client = configuration.newClient(configuration.writeConcernMax(), { poolSize: 1 });
+      var client = configuration.newClient(configuration.writeConcernMax(), {
+        poolSize: 1
+      });
```

Comment:
Why are these changing if the code is the same? Maybe double check prettier version?

Reply:
fixed!